jwst/regtest/test_nirspec_ifu.py

```diff
@@ -49,34 +100,52 @@ def run_spec3(jail, rtdata_module):
         }
     }
 
-    return rt.run_step_from_dict(rtdata_module, **step_params)
+    rtdata = rt.run_step_from_dict(rtdata, **step_params)
+    return rtdata
 
 
 @pytest.mark.bigdata
 @pytest.mark.parametrize(
     'suffix',
-    ['assign_wcs', 'cal', 'flat_field', 'imprint_subtract', 'msa_flagging', 'pathloss', 'photom', 's3d', 'srctype', 'x1d']
```

Comment:
So has testing of the ``imprint`` step been removed from the test? This was added recently in order to fill a hole in NIRSpec testing. It requires running the ``calwebb_spec2`` pipeline with an ASN as input, which includes the imprint image as one of its members. And of course setting ``steps.imprint_subtract.save_results=true``.

Reply:
Aaaaannnnnnnddddd this would be the case of getting data stomped on. Will restore.

Reply:
Pushed. One more try...